lib/Game.rb

```diff
@@ -0,0 +1,54 @@
+require_relative "Player"
+require_relative "Board"
+
+class Game
+    def initialize
+        @board =  Board.new
```

Comment:
Why did we need to initialize Board inside Game? Isn't it possible to just pass an instantiated Board object into game?